### PR TITLE
Simplify css included with project

### PR DESCRIPTION
**Pull request by neilff**
- CSS had some oddities, like the `overflow: hidden`

### CODE REVIEW

**Comment by JaKXz** [comment]
Just curious, how are /were you using `@import 'file'` in `.css` files?

**Comment by neilff** [comment]
I didn't set up the css, but I believe it is through Post CSS @winkerVSbecks or @mattmischuk would be better resources to ask about that

**Comment by winkerVSbecks** [comment]
Using post-css and cssnext: `style-loader!css-loader!postcss-loader!cssnext-loader`

Just `style-loader!css-loader` allow you to import `css` files. `postcss-loader!cssnext-loader` let you compile CSS4 syntax and auto-prefixer, etc.

File: src/styles/base.css [hunk 1]
```diff
@@ -9,5 +9,4 @@ body {
   margin: 0px;
   padding: 0px;
   -webkit-font-smoothing: antialiased;
```
Comment (mattmischuk) [inline src/styles/base.css]: While your here we should probably remove this line as well.

**Pull request merged**